### PR TITLE
Fix: LT-19083 When "Repair" resets ProjectsDir to C:\

### PR DESCRIPTION
**Pull request by samdoss**
* Included the project registry to get the setting value

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/genericinstaller/27)

### CODE REVIEW

File: BaseInstallerBuild/Framework.wxs [hunk 1]
```diff
@@ -98,7 +101,7 @@
 		DllEntry="VerifyDataDirPath"  />
 	<SetProperty Id='REGISTRYKEY' Value='$(var.SafeApplicationName)\\$(var.MajorVersion)'  Before="AppSearch">not REGISTRYKEY</SetProperty>
 	<SetProperty Id='APPFOLDERREGSZNAME' Value='Program_Files_Directory_$(var.SafeApplicationName)'  Before="AppSearch">not APPFOLDERREGSZNAME</SetProperty>
-	<SetProperty Id='DATAFOLDERREGSZNAME' Value='Settings_Directory'  Before="AppSearch">not DATAFOLDERREGSZNAME</SetProperty>
+	<SetProperty Id='DATAFOLDERREGSZNAME' Value='[DATAFOLDERREGSZNAME]'  Before="AppSearch">not DATAFOLDERREGSZNAME</SetProperty>
```
[the remark this answers is not in the document]
Comment (samdoss) [reply BaseInstallerBuild/Framework.wxs]: This change is not required. I will revert this change.

File: BaseInstallerBuild/Framework.wxs [hunk 2]
```diff
@@ -88,6 +88,9 @@
 	<Property Id="GALATIA_INSTALLED" Value='False'/>
 	<Property Id="EZRA_INSTALLED" Value='False'/>-->
 
+	<Property Id="DATAFOLDER" Secure='yes'>
+	  <RegistrySearch Id='RegistryDataBool' Root='HKLM' Key='SOFTWARE\[REGISTRYKEY]' Name='[DATAFOLDERREGSZNAME]' Type='raw'/>
```
[the remark this answers is not in the document]
Comment (samdoss) [reply BaseInstallerBuild/Framework.wxs]: ok. I will change it to the meaning full name.

**Comment by jasonleenaylor** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io/reviews/sillsdev/genericinstaller/27#-:-LC5AbSF5LvhvUaQgQtA:bnfp4nl)*

**Comment by jasonleenaylor** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io/reviews/sillsdev/genericinstaller/27)*

**Pull request merged**